lms/djangoapps/courseware/toggles.py

```diff
@@ -83,6 +83,34 @@
     f'{WAFFLE_FLAG_NAMESPACE}.disable_navigation_sidebar_blocks_caching', __name__
 )
 
+# .. toggle_name: courseware.disable_navigation_sidebar
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Disable navi sidebar on Learning MFE
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2024-03-07
+# .. toggle_target_removal_date: None
+# .. toggle_tickets: AXIMST-611
+# .. toggle_warning: None.
+COURSEWARE_MICROFRONTEND_SIDEBAR_DISABLED = CourseWaffleFlag(
+    f'{WAFFLE_FLAG_NAMESPACE}.disable_navigation_sidebar', __name__
+)
+
+# .. toggle_name: courseware.show_default_right_sidebar
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
```

Comment:
```suggestion
# .. toggle_default: True
```

Reply:
@arbrandes I have applied this change, but now I'm thinking if we would actually want to enable`always_open_auxiliary_sidebar` by default?